nextjs/src/app/reports/[id]/(components)/MapLayers/PoliticalChoropleths.tsx

```diff
@@ -114,60 +114,63 @@ const PoliticalChoropleths: React.FC<PoliticalChoroplethsProps> = ({
           layout={{ visibility, 'line-join': 'round', 'line-round-limit': 0.1 }}
         />
       </Source>
-      <Source
-        id={`${tileset.mapboxSourceId}-area-count`}
-        type="geojson"
-        data={getAreaGeoJSON(dataByBoundary)}
-      >
-        <Layer
+      {showBoundaryNames && (
```

Comment:
Again, let's change the `visibility` variable rather than conditionally rendering a layer.

Reply:
should be done now